<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>150</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Valores en el tiempo :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="incertidumbre_volt">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>200</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="incertidumbre_top">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>280</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val_frecuencia">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>160</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>120</y>
      <width>153</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="velocidad">
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>velocidad :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="frecuencia">
       <property name="text">
        <string>frecuencia :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="voltaje">
       <property name="text">
        <string>voltaje :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="temperatura">
       <property name="text">
        <string>temperatura :</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="tiempo_operacion">
       <property name="text">
        <string>tiempo de operacion :</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="color_frec">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>65</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="color_vel">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>120</y>
      <width>65</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="color_temp">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>240</y>
      <width>65</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="ingresar_carnet">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>251</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ingresar numero del carnet :</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="carnet">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>90</y>
      <width>251</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="color_top">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>280</y>
      <width>65</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="incertidumbre_frec">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>160</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val_voltaje">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>200</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="incertidumbre_temp">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>240</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="registro">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>170</y>
      <width>251</width>
      <height>281</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="incertidumbre_vel">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>120</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="color_volt">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>200</y>
      <width>65</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(211, 215, 207);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="valores">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>30</y>
      <width>181</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Actualizar valores</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="val_toperacion">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>280</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val_temperatura_2">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>240</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="val_velocidad">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>120</y>
      <width>121</width>
      <height>30</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
